dojo/templates/dojo/engagement.html

```diff
@@ -60,8 +60,15 @@ <h3 class="has-filters">
                                 <td>
                                     {% for e in p.engagement_set.all %}
                                         {% if e.active %}
-                                            <a class="eng_link" href="{% url 'view_engagement' e.id %}">
+						<div class="lineContainer">
+							<a  style="display: inline" class="eng_link" href="{% url 'view_engagement' e.id %}">
                                                 {% if e.name %}{{ e.name }} {% endif %}{{ e.target_start }}</a>
+							| Lead : {{ e.lead.first_name }}
+                                                        |
+                                                        {%  for test in e.test_set.all %}
+                                                                <a href="{% url 'view_test' test.id %}"> {{ test.test_type }} : {{ test.lead.first_name }} </a> |
```

Comment:
Maybe only add lead info if lead is set (if lead is not a required field, see other comment)